### PR TITLE
Navbar/Footer Icons complete

### PR DESCRIPTION
**Pull request by CMeier556**
Footer is pressed against navbar because there is no content in-between. Responsive anchor tags have yet to be added.

### CODE REVIEW

File: index.html
Comment (Joshsilas) [inline index.html:1]: No title on HTML head

Comment (Joshsilas) [inline index.html:1]: Apart from that Works well on my browser, the changes in screen size are responsive and the code is neat. Be good to see how it looks with content inside.

**Review by Joshsilas — CHANGES_REQUESTED** [review]
No title on HTML head

Apart from that Works well on my browser, the changes in screen size are responsive and the code is neat. Be good to see how it looks with content inside.

File: index.html
Comment (CMeier556) [reply index.html:1]: Done

Comment (CMeier556) [reply index.html:1]: html tag closed

**Review by Joshsilas — APPROVED** [review]
all good

**Pull request closed without merging**